### ISSUE TITLE
Version 2.5.8 has wrong/longer width with it's time-picker

### ISSUE THREAD

**Issue by alkunjo** [fooloomanzoo/datetime-picker#44]
I recently use the latest version 2.5.8 of datetime-picker. Somehow when I use it to my code which I'll attach on the below, it shows longer width. Is there anything I can do?
```
<time-picker clamp="second" id="mondayOpeningHour" confirmed-datetime="2018-01-01T09:00:00.000"></time-picker>
```
![screenshot from 2018-03-26 13-56-44](https://user-images.githubusercontent.com/9002652/37891215-a1fba8ce-30fd-11e8-909a-7ae233f0e490.png)

**Comment by fat-crienturu** [comment]
I also have the issue. 
It's caused by input-picker-pattern version 2.2.9. If I downgrade input-picker-pattern to 2.2.8 then it works.

**Comment by alkunjo** [comment]
Ups, sorry seems I was mistaken naming the latest version, which I mean is version 2.8.5. The problem is that I cannot specify to downgrade to certain version like 2.5.5 or to 2.5.11. Even I do not have the `input-picker-pattern` on my bower.json so I cannot change the dependency directly to 2.2.8 and reinstall it like you said. Every time I try to install the lower versions of `datetime-picker` like 2.5.5, 2.5.11, 2.6.1, 2.7.0 it always install all dependency to the latest version  it always installed the 2.8.5 version like below:

```
├─┬ datetime-picker#2.8.5
│ ├─┬ datetime-input#2.3.1
│ │ ├─┬ input-picker-pattern#2.2.9
│ │ │ ├── iron-overlay-behavior#2.3.3
│ │ │ └── polymer#2.6.0
│ │ ├─┬ number-input#2.3.7
│ │ │ ├─┬ input-picker-pattern#2.2.9
│ │ │ │ ├── iron-overlay-behavior#2.3.3
│ │ │ │ └── polymer#2.6.0
│ │ │ ├── polymer#2.6.0
│ │ │ ├── property-mixins#2.4.1
│ │ │ └─┬ text-input#2.0.6
│ │ │   ├─┬ input-picker-pattern#2.2.9
│ │ │   │ ├── iron-overlay-behavior#2.3.3
│ │ │   │ └── polymer#2.6.0
│ │ │   └── polymer#2.6.0
│ │ ├── polymer#2.6.0
│ │ └── property-mixins#2.4.1

```
Is it the mistake on my `bower` or is there anything else I can do?

**Comment by fooloomanzoo** [comment]
It should be fixed with the new version `2.8.7`. Please consider the new way of [styling](https://github.com/fooloomanzoo/datetime-picker#styling).
